### PR TITLE
Redmine #7961: Acceptance test to make sure def.json:inputs is picked up.

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by tzz** [comment]
looks OK to me

File: tests/acceptance/00_basics/def.json/inputs.cf [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+      "" usebundle => file_copy("$(this.promise_filename).json", "$(sys.inputdir)/def.json");
+}
+
+#######################################################
+
+bundle agent check
+{
+  vars:
+    "command" string => "$(sys.cf_promises) --show-classes -f $(sys.inputdir)/promises.cf|$(G.grep) test_class";
```
Comment (nickanderson) [inline tests/acceptance/00_basics/def.json/inputs.cf]: I think you can use `dcs_passif_output` here instead.

Comment (nickanderson) [inline tests/acceptance/00_basics/def.json/inputs.cf]: oh duh you did, ignore me.

**Comment by kacf** [comment]
Cherry-picked to 3.8.x, 3.7.x is pending.

**Pull request merged**